openverse_catalog/dags/providers/provider_api_scripts/finnish_museums.py

```diff
@@ -301,6 +311,18 @@ def _get_image_url(img, image_url=API_URL):
             return None
         return image_url + img
 
+    @staticmethod
+    def get_creator(obj):
+        authors = []
+        for author_type in ["primary", "secondary", "corporate"]:
+            author = obj.get("authors", {}).get(author_type)
+            if author is None or type(author) != dict:
+                continue
+            author = "; ".join(list(author.keys()))
+            authors.append(author)
+
+        return "; ".join(authors)
```

Comment:
I _think_ this will return an empty string if no authors are found, but it might be better to return `None` in that case to have `NULL` in the database rather than an empty string. Maybe we could do `self.get_creator(data) or None` above?

Reply:
To make sure that we don't have different implementations for when creator name is not available (`""` vs `None`), we could add a check and conversion here:
https://github.com/WordPress/openverse-catalog/blob/4c07e63181096a4e9925c103e5480f2b0bf36107/openverse_catalog/dags/common/storage/columns.py#L397
@AetherUnbound, should we create an issue for that?

Reply:
Good catch! 😮 I was naive to believe it would return `None` this way. Fixed in last commits. Thanks, @AetherUnbound.

@obulat Adding an extra check there or/and in the `MediaStore` classes sounds good!

Reply:
Your idea about adding a check in `MediaStore` is better than the columns one, @krysal. We could add 
```py
if title === "":
  title = None
```
in the MediaStore here:
https://github.com/WordPress/openverse-catalog/blob/9987322f10c91e02d65469d394455d7d1bc3a231/openverse_catalog/dags/common/storage/media.py#L93

Are there any other string fields that need that, too?

Reply:
I think `title` and `creator` are both good fields to check for that!